src/main/webapp/data/userMapData.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lastUpdated": "31 May 2017",
+  "lastUpdated": "9 July 2017",
```

Comment:
This should be `30 June`. To minimize future errors, we can in fact use `End of June 2017`